doc/api/tty.md

```diff
@@ -77,7 +77,7 @@ Note that `CTRL`+`C` will no longer cause a `SIGINT` when in this mode.
 added: v0.5.8
 -->
 
-The `tty.WriteStream` class is a subclass of `net.Socket` that represents the
+The `tty.WriteStream` class is a subclass of [`net.Socket`][] that represents the
```

Comment:
This line has 81 characters low which violates 80 characters limit and will cause doc linting issue. Can you rewrap?